editor/editor_node.cpp

```diff
@@ -970,6 +957,18 @@ void EditorNode::_sources_changed(bool p_exist) {
 			load_scene(defer_load_scene);
 			defer_load_scene = "";
 		}
+
+		// Only enable addons once resources have been imported
+		_initializing_addons = true;
+		Vector<String> addons;
+		if (ProjectSettings::get_singleton()->has_setting("editor_plugins/enabled")) {
+			addons = ProjectSettings::get_singleton()->get("editor_plugins/enabled");
+		}
+
+		for (int i = 0; i < addons.size(); i++) {
+			set_addon_plugin_enabled(addons[i], true);
+		}
+		_initializing_addons = false;
```

Comment:
Won't this mean that it will re-initialize addons every time sources change, instead of only once at startup?

Reply:
I don't think so, because this code will only run when `waiting_for_first_scan` is true.

Reply:
Ah right, I misread as GH messes tab width for me between near and extended diff.